frontend/src/components/pages/CampOverview/ViewCamperModal.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
```

Comment:
Could you also move all the ViewCamperModal specific files to a subfolder called "ViewCamperModal" under tthe CampOverviewPage folder? Also rename this main modal component file to `index.tsx` as that is more consistent with our codebase

Reply:
Done.